src/fixate/_switching.py

```diff
@@ -104,16 +169,16 @@ def __init__(self, update_pins: Optional[PinUpdateCallback] = None):
         else:
             self._update_pins = update_pins
 
+        self._pin_set = frozenset(self.pin_list)
```

Comment:
Not sure why this has moved around, but not the is in the wrong place. It should probably just move back where it was.

Reply:
I moved this in the previous commit due to playing around with some things. It should be back to where it was originally now.